src/OpenFlight/Books/Domain/SeatValueObject.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+
+namespace CodelyTv\OpenFlight\Books\Domain;
+
+
+class SeatValueObject
+{
+
+    const class_types = ['first', 'business'];
+
+    private int $number;
+    private string $letter;
+    private string $class;
+
+
+    public function __construct(int $number, string $letter, string $class)
+    {
+        $this->number = $number;
+        $this->letter = $letter;
+        $this->class = $class;
+    }
+
+    public static function createSeat(int $number, string $letter, string $class): SeatValueObject
+    {
+        self::validateNumber($number);
+        self::validateLetter($letter);
+        self::validateClass($class);
+        return new self($number, $letter, $class);
+    }
+
+    private static function validateNumber(int $number)
+    {
+        if ($number <= 0 || $number === 13) {
```

Comment:
Todas las constantes deben ir definidas en una variable al menos
```suggestion
        if ($number <= self::MINIMUM_SEAT || $number === self::MAXIMUM_SEAT) {
```